### ISSUE TITLE
MPB indices

### ISSUE THREAD

**Issue by mlubin** [JuliaOpt/JuMP.jl#468]
We don't currently provide any well-defined way to map from MPB variable indices into JuMP variables. This seems to be a big gap in the abstraction. Look at the IIS example:

```
    println("Irreducible Inconsistent Subsystem (IIS)")
    println("Variable bounds:")
    for i in 1:numvar
        v = Variable(m, i)
        if iislb[i] != 0 && iisub[i] != 0
            println(getLower(v), " <= ", getName(v), " <= ", getUpper(v))
        elseif iislb[i] != 0
            println(getName(v), " >= ", getLower(v))
        elseif iisub[i] != 0
            println(getName(v), " <= ", getUpper(v))
        end
    end
```

What about exporting a function which takes a model and MPB variable index and returns a corresponding JuMP variable?

**Comment by mlubin** [comment]
(Title changed since I changed my mind halfway through writing up the issue.)

**Comment by mlubin** [comment]
What if we guarantee and document that ``Variable(m, idx)`` is valid when ``idx`` is an MPB index? If we ever need to use a nontrivial map, we can make the appropriate internal changes to keep ``Variable(m,idx)`` working.

**Comment by mlubin** [comment]
Same for ``ConstraintRef{LinearConstraint}`` also

**Comment by mlubin** [comment]
I think this is resolved by #479 and https://github.com/JuliaOpt/JuMP.jl/commit/794c91157de5d4b2917ce5a27b6a1a264f1f9d3a
